index.html

```diff
@@ -76,132 +82,117 @@ <h3 class="H-title">I'am a Software Developer</h3>
-                      <p class="papd-p">Profesional Art Printing Data
-                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-                        <ul class="language-button1">
-                          <li>html</li>
-                          <li>bootstrap</li>
-                          <li>Ruby</li>
-                        </ul>
-                        <button 
-                        type="button"
-                        class="button2">See Project</button>
-                    </div>
-                    <div class="card-work">
-                      <h1 class="papd">Professional Art<br>Printing Data</h1>
-                      <p class="papd-p">Profesional Art Printing Data
-                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-                        <ul class="language-button1">
-                          <li>html</li>
-                          <li>bootstrap</li>
-                          <li>Ruby</li>
-                        </ul>
-                        <button 
-                        type="button"
-                        class="button2">See Project</button>
-                    </div>
-                     <div class="card-work">
-                      <h1 class="papd">Professional Art<br>Printing Data</h1>
-                      <p class="papd-p">Profesional Art Printing Data
-                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
-                        <ul class="language-button1">
-                          <li>html</li>
-                          <li>bootstrap</li>
-                          <li>Ruby</li>
-                        </ul>
-                      <button 
-                      type="button"
-                      class="button2">See Project</button>
-                    </div>
-              </section>
+      <img
+      src="./images/Img.png"
+      class="work__image"
+      alt="project works"
+      data-img="image"
+    />
+      <div class="work__details">
+        <div>
+          <h3 class="work__details__title">Multi-Post Stories</h3>
+        </div>
+       <div>
+        <p class="work__details__p">A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text
+          ever since the 1500s, when an unknown printer took a standard
+          dummy text. A daily selection of privately personalized reads; no
+          accounts or sign-ups required.</p>
+
+       </div>
+       <div>
+        <ul class="single_work_stack">
+          <li>CSS</li>
+          <li>html</li>
+          <li>Bootstrap</li>
+          <li>Ruby</li>
+      </ul>
+       </div>
+       <div>
+         
+         <button type="button">See project</button>
+       </div>
```

Comment:
Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_9](https://user-images.githubusercontent.com/69638013/161400287-eeba0b53-4215-48b3-bc80-25182d56ddc1.png)  |  ![Screenshot_8](https://user-images.githubusercontent.com/69638013/161400283-fa91d5f2-ae81-47a0-9faf-7ade39ee7281.png)

- [ ] Kindly add the right background color and border outline for the li tags
- [ ] Please add more spacing between the li tags and the see project button

Reply:
- [ ] As suggested by the previous reviewer, kindly change the background colour according to the Figma design. Make sure to change the background colour to #fff.

Reply:
- [ ] As required by the previous reviewer, remove the background color to your `languages` and instead, add a border CSS property to it.

- [ ] Don't forget to add margin-top or padding-top to your `see project` button so as to separate it with the languages

- Figma design:
![Screenshot1](https://user-images.githubusercontent.com/79058364/161404445-80c98db9-6550-4ae4-b9fe-24988717ba86.png)

- Your design

![Screenshot11](https://user-images.githubusercontent.com/79058364/161404455-a95b23d6-3062-44de-beed-dd7e68844fca.png)